### ISSUE TITLE
trigger rate and channel rate do not match if used in single trigger mode with one detector

### ISSUE THREAD

**Issue by musner** [achim1/muonic#58]
when only connecting one scintillation plate (e.g. to channel 0) and setting the trigger to single, one would expect that the channel 0 rate and the trigger rate match exactly. This is the case for low rates but there are mismatches if the rates are comparably high (>100Hz).

**Comment by hpbretz** [comment]
Rates of >100Hz should be cases where the Threshold is set too low and the rate is mostly noise. Maybe the DAQ card gives a wrong count in these cases. Is the trigger rate too low or too high?

**Comment by musner** [comment]
Rates > 100 Hz was actually a false statement from memory. It was more around 20-30 Hz. I attached a plot. The trigger rate was too small. It did seem to happen with a defect PMT though, so it might have been a hardware problem. But even if the PMT was unstable, should this mismatch really occur?

![messungkamiokanne2](https://cloud.githubusercontent.com/assets/6197530/9194071/dcdcf872-4018-11e5-968d-fa91ac553f38.png)

**Comment by hpbretz** [comment]
I don't think it should, but this appers to be a problem of the DAQ, not muonic. In prinicple, muonic just asks the daq every 5s to give a count of the channels and the trigger, substracts the last count for each value and divides it by the time difference. If this happens again or can be reproduced, I would look into it again, otherwise chalk this up to a problem with the DAQ.
